README.rst

```diff
@@ -1969,79 +1969,69 @@ You can also use the ``alert_field`` key in metric configuration
 which allows ``AlertSettings`` to check the ``threshold`` on
 ``alert_field`` instead of the default ``field_name`` key.
 
-In the following example, we added ``alert_field`` to the metric configuration
-for the iperf metric in order to check the threshold on **iperf bandwidth**.
+**Note**: It will raise ``ImproperlyConfigured`` exception if a metric configuration
+is already registered with same name (not to be confused with verbose_name).
+
+If you don't need to register a new metric but need to change a specific key of an
+existing metric configuration, you can use `OPENWISP_MONITORING_METRICS <#openwisp_monitoring_metrics>`_.
+
+The following example shows how to use the
+`setting <https://github.com/openwisp/openwisp-monitoring#openwisp_monitoring_metrics>`_
+to reconfigure the system for `iperf check <#iperf-1>`_ to send an alert if
+the measured **TCP bandwidth** has been less than **10 Mbit/s** for more than **2 days**.
+
+1. Iperf check doesn't have any ``AlertSettings`` by default,
+but we can easily create one through the OpenWISP admin interface.
```

Comment:
1. By deafult, Iperf checks do not have any alert settings defined,
but it is easy to create one through the admin interface as shown below.